core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -2161,6 +2161,96 @@ class DAGSchedulerSuite extends SparkFunSuite with LocalSparkContext with Timeou
     }
   }
 
+  test("[SPARK-19263] DAGScheduler should avoid sending conflicting task set") {
```

Comment:
Can we make this test name a bit more specific, maybe "DAGScheduler should not submit multiple active tasksets, even with late completions from earlier stage attempts" (or if you have another suggestion)